src/iiif/object_store.py

```diff
@@ -45,7 +45,10 @@ def create_object_store_temp_url(connection, file_name, expiry_minutes=0, expiry
     return url
 
 
-def remove_old_zips_from_object_store():
+def remove_old_zips_from_object_store(logger=None):
+    if logger:
+        log = logger
```

Comment:
Dit werkt alleen als je de logger parameter meegeeft. Als je hem weglaat krijg je een `UnboundLocalError` omdat hij probeert de lokale `log` te gebruiken, en niet de globale. Die is dus vervolgens unbound als je geen parameter meegeeft:
 
```python
Python 3.8.10 (default, Jun  2 2021, 10:49:15)                            
[GCC 9.4.0] on linux                                                      
Type "help", "copyright", "credits" or "license" for more information.    
>>> from logging import getLogger                                         
>>> def remove_old_zips_from_object_store(logger=None):                   
...   if logger:                                                          
...     log = logger                                                      
...   log.info('something')                                               
...                                                                       
>>> remove_old_zips_from_object_store()                                   
Traceback (most recent call last):                                        
  File "<stdin>", line 1, in <module>                                     
  File "<stdin>", line 4, in remove_old_zips_from_object_store            
UnboundLocalError: local variable 'log' referenced before assignment      
```

Ik zou het omdraaien, zodat `logger` wordt op de globale `log` gezet als hij niet is meegeven, en vervolgens `logger` gebruiken om bericht te sturen...

```python
def remove_old_zips_from_object_store(logger=None):
    if logger is None:
        logger = log
    ...
    logger.info('something')
```